### DISCUSSION TITLE
This is probably a KDE issue: the alt key works weird

### DISCUSSION THREAD

**Discussion by shinarit** [doublecmd/doublecmd#1985] [General]
On my Ubuntu 22 with KDE, I set up alt for quick searching through files, and this overrides the normal window behavior when holding alt. Pressing alt focuses the menu as usual.
On Kubuntu 24, pressing alt does nothing, but holding alt activates both the general window functionality (so shortcut to menu options) and the quick search, with the former having precedence. So alt+t for example opens the tabs menu, but alt+. will search for files with . in their name.

I don't think this is a bug in DBC, or even in KDE, because no other window behaves like this, most likely just some incompatibility with some newer features and options. But I'm lost, I went through both plasma and DBC settings, and have no idea how to resolve this. Not a bug, but very annoying.

**Comment by alexx2000** [comment]
#1327
